neo4j-jdbc/src/main/java/org/neo4j/jdbc/Neo4jDatabaseMetaData.java

```diff
@@ -41,6 +41,8 @@
 
 	public static final Logger LOGGER = Logger.getLogger(Neo4jDatabaseMetaData.class.getName());
 
+	public static final String GET_DBMS_FUNCTIONS = "CALL dbms.functions() YIELD name RETURN name ORDER BY name ASC";
```

Comment:
@fbiville Do you think makes sense filter the functions for only the ones bundled with the database?

Reply:
I mean with this query we catch also customs like apoc...

Reply:
That's a good point.
Actually, we may want to filter out any user-defined functions (like the standard ones APOC and/or the user provides, the custom-defined from APOC).
Filtering out the custom should be easy as the prefix is... fixed.

However, I do not see a way to distinguish built-in functions from user-defined ones. Built-in functions do not have to be namespaced (contrary to user-defined functions), but date/time built-in functions are namespaced.

Reply:
yep you're right